tsl/test/t/006_job_crash_log.pl

```diff
@@ -0,0 +1,65 @@
+# This file and its contents are licensed under the Timescale License.
+# Please see the included NOTICE for copyright information and
+# LICENSE-TIMESCALE for a copy of the license.
+
+use strict;
+use warnings;
+use PostgresNode;
+use Test::More;
+use TestLib;
+
+my $node = get_new_node('primary');
+$node->init(allows_streaming => 1);
+$node->append_conf('postgresql.conf',
+	"shared_preload_libraries = 'timescaledb'");
+$node->start;
```

Comment:
These steps are already done if you create a [`TimescaleNode`](https://github.com/timescale/timescaledb/blob/main/test/perl/TimescaleNode.pm) instead of a `PostgresNode`. It will also make sure that you do all initialization that are necessary, now and in the future.